plenario/api/condition_builder.py

```diff
@@ -20,80 +21,78 @@
 }
 
 
-def parse_general(table, field, value):
```

Comment:
In order to streamline the process of specifying column conditions, I've chosen to convert a user's request args into a dataset__filter (if they didn't choose to include one already). This change removes the need for two different ways to parse user arguments, and allowed me to remove parse_general.